mmdet/visualization/local_visualizer.py

```diff
@@ -158,22 +159,59 @@ def _draw_instances(self, image: np.ndarray, instances: ['InstanceData'],
         if 'masks' in instances:
             labels = instances.labels
             masks = instances.masks
+            if isinstance(masks, torch.Tensor):
+                masks = masks.numpy()
+
             max_label = int(max(labels) if len(labels) > 0 else 0)
             mask_color = palette if self.mask_color is None \
                 else self.mask_color
             mask_palette = get_palette(mask_color, max_label + 1)
             colors = [mask_palette[label] for label in labels]
 
+            text_palette = get_palette(self.text_color, max_label + 1)
+            text_colors = [text_palette[label] for label in labels]
+
             polygons = []
             for i, mask in enumerate(masks):
                 contours, _ = bitmap_to_polygon(mask)
                 polygons.extend(contours)
             self.draw_polygons(polygons, edge_colors='w', alpha=self.alpha)
             self.draw_binary_masks(masks, colors=colors, alphas=self.alpha)
 
+            if 'bboxes' not in instances or instances.bboxes.sum() == 0:
```

Comment:
Does `instances.bboxes.sum() == 0` represent dummy bboxes? May add some comments.

Reply:
consider use .numel() for more clear usages.

Reply:
OK